README.md

```diff
@@ -336,7 +311,34 @@ curl -u "$username":"$password" \
 
 ### 11. Run the application
 
-## Pre-requisite
+Use the ``Deploy to IBM Cloud`` button **OR** run locally.
+
+### Deploy to IBM Cloud
+
+[![Deploy to Bluemix](https://bluemix.net/deploy/button.png)](https://bluemix.net/deploy?repository=https://github.com/IBM/sms-analysis-with-wks)
+
+1. Press the above ``Deploy to IBM Cloud`` button and then click on ``Deploy``.
+
+2. In Toolchains, click on Delivery Pipeline to watch while the app is deployed. Once deployed, the app can be viewed by clicking 'View app'.
+
+![](doc/source/images/toolchain-pipeline.png)
+
+3. To see the app and services created and configured for this journey, use the IBM Cloud dashboard. The app is named `sms-analysis-with-wks` with a unique suffix. The following service is created:
+    * sms-nlu-service
+
+4. One last step is required to configure the app to run with your WKS model.
+
+In the IBM Cloud dashboard, find the app that was created. Click on `Runtime` on the menu and navigate to the `Environment variables` tab.
```

Comment:
Probably mention that you click on the app to get into manage before you can find Runtime (?)

Reply:
can do